### DISCUSSION TITLE
Remove filter icon from column filter input

### DISCUSSION THREAD

**Discussion by gaspardip** [KevinVandy/material-react-table#53] [Q&A] [answered]
Currently doing it like 

```ts
        MuiTableCell: {
          styleOverrides: {
            root: {
              '& .MuiInputBase-root > .MuiSvgIcon-root': {
                display: 'none',
              },
            },
          },
        },
```

but I was wondering if there was a way to avoid rendering it to the DOM altogether

**Comment by KevinVandy** [comment]
You can override the mui InputProps on the TextField component entirely if you want.

```ts
muiTableHeadCellFilterTextFieldProps={{
  InputProps: {
    startAdornment: null,
    endAdornment: null,
}}
```

Problem is you also remove the endAdornment too.

**Reply by gaspardip** [reply]
That would be the "Clear Filter" icon right? If so, definitely not ideal

**Reply by KevinVandy** [reply]
then I think your solution is already ideal. I guess I could think over the design if no filter modes are enabled. I was following [material-table's](https://material-table.com/#/docs/features/filtering) design for the filters, but if there is a placeholder, maybe we won't show the icon by default

**Comment by KevinVandy** [comment, marked as answer]
After re-considering the default design, as of `v0.41.0`, the start icon will only show when `enableColumnFilterModes` is enabled

**Reply by gaspardip** [reply]
That works, thanks!
